diff --color -ruN ../linux-live-2.6.orig/bootfiles/syslinux.cfg ./bootfiles/syslinux.cfg
--- ../linux-live-2.6.orig/bootfiles/syslinux.cfg	2022-05-20 23:38:24.515627421 +0200
+++ ./bootfiles/syslinux.cfg	2022-05-22 17:04:57.344435970 +0200
@@ -8,11 +8,11 @@
 
 LABEL default
 MENU LABEL Run Linux
-KERNEL /boot/vmlinuz
-APPEND vga=769 initrd=/boot/initrfs.img load_ramdisk=1 prompt_ramdisk=0 rw printk.time=0 apparmor=0
+KERNEL /boot/vmlinuz-virt
+APPEND initrd=/boot/initrfs.img load_ramdisk=1 prompt_ramdisk=0 rw printk.time=0 apparmor=0
 
 LABEL default
 MENU LABEL Run Linux debug
-KERNEL /boot/vmlinuz
-APPEND vga=769 initrd=/boot/initrfs.img load_ramdisk=1 prompt_ramdisk=0 rw printk.time=0 apparmor=0 debug
+KERNEL /boot/vmlinuz-virt
+APPEND initrd=/boot/initrfs.img load_ramdisk=1 prompt_ramdisk=0 rw printk.time=0 apparmor=0 debug
 
diff --color -ruN ../linux-live-2.6.orig/config ./config
--- ../linux-live-2.6.orig/config	2022-05-20 23:38:24.519627207 +0200
+++ ./config	2022-05-20 23:39:05.265547834 +0200
@@ -17,7 +17,7 @@
 # Kernel file, will be copied to your Live Kit
 # Your kernel must support aufs and squashfs. Debian Jessie's kernel is ready
 # out of the box.
-VMLINUZ=/vmlinuz
+VMLINUZ=/boot/vmlinuz-virt
 
 # Kernel version. Change it to "3.2.28" for example, if you are building
 # Live Kit with a different kernel than the one you are actually running
